### ISSUE TITLE
Problem: Missing implementation for communicating between Core operation and Guest wrapper

### ISSUE THREAD

**Issue by foreseaz** [nleak/nleak#49]
A possible solution, NodeJS event IPC:

```
// in wrapper.js
// $$$EVENT$$$
on("$$$EVENT$$$", (data: any)) => {
  data:{ 
    fn_name: "$$$INSTRUMENT_PATHS$$$",
    args: {} as any,
  }

  switch (data.fn_name) {
    case "$$$INSTRUMENT_PATHS$$$":
      $$$INSTRUMENT_PATHS$$$(args.trees);
    case
})

```

**Comment by foreseaz** [comment]
<img width="1221" alt="image" src="https://user-images.githubusercontent.com/5697641/200959674-078542e7-0cfe-418e-a9de-6def250c6a04.png">

**Comment by foreseaz** [comment]
Thanks @Frost-Lee , closed by #52
